runtime/doc/intro.txt

```diff
@@ -193,6 +193,16 @@ Elvis	Another Vi clone, made by Steve Kirkendall.  Very compact but isn't
 
 Vim	Nvim is based on Vim.  https://www.vim.org/
 
+==============================================================================
+Bram Moolenaar                                    *Bram* *brammool*
+
+Nvim is a text editor built as a continuation of Vim, Vi Improved.  Vim was
```

Comment:
I also don't think "continuation" is the right word here.

Reply:
"fork" is how we usually describe it